Description: Add "Ubuntu" to the user-agent text.
Author: Chad Miller <chad.miller@canonical.com>,
  Joachim Bauch <bauch@struktur.de>

--- a/content/common/user_agent.cc
+++ b/content/common/user_agent.cc
@@ -192,7 +192,7 @@
   std::string user_agent;
   base::StringAppendF(
       &user_agent,
-      "Mozilla/5.0 (%s) AppleWebKit/%d.%d (KHTML, like Gecko) %s Safari/%d.%d",
+      "Mozilla/5.0 (%s) AppleWebKit/%d.%d (KHTML, like Gecko) Ubuntu %s Safari/%d.%d",
       os_info.c_str(),
       WEBKIT_VERSION_MAJOR,
       WEBKIT_VERSION_MINOR,
